src/routes/pricing/+page.svelte

```
@@ -1,58 +1,488 @@
+						</li>
+					</ul>
 				</div>
-				<div class="mt-6 text-3xl font-extrabold">$0</div>
-				<div class="card-actions mt-6">
-					<a class="btn btn-primary" href="/sign-up">Get started</a>
+			</div>
+		</div>
+
+		<div class="text-center">
+			<p class="text-base-content/60 mb-4">Ready to get started?</p>
+			<a href="/sign-up" class="btn btn-primary">Create an account</a>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)